examples/preact/src/hotLoaderSetup.js

```diff
@@ -1,4 +1,34 @@
-import reactHotLoader from 'react-hot-loader'
-import preact from 'preact'
+import reactHotLoader, { compareOrSwap } from 'react-hot-loader'
+import preact, { setComponentComparator } from 'preact'
 
 reactHotLoader.inject(preact, 'h')
+
+setComponentComparator(compareOrSwap)
+//(oldC, newC) => compareOrSwap(oldC, Object.getPrototypeOf(newC)));
+
+// changes to preact
+
```

Comment:
This is changes to be added into Preact code base. Literraly - just wrapping `node._componentConstructor == vnode.nodeName` by a controllable function.